README.md

```diff
@@ -204,27 +208,23 @@ make update-vendor
 
 #### Debugging dependencies
 
-Why is a specific package imported: `go mod why $pkgname`.
+Why is a specific package imported?
+
+`go mod why $pkgname`
+
+Why is a specific module imported?
+
+`go mod why -m $modname`
 
-Why is a specific module imported: `go mod why -m $modname`.
+Why is a specific version of a module imported?
 
-Why is a specific version of a module imported: `go mod graph | grep $modname`.
+`go mod graph | grep $modname`
 
 ## Why did We Build Teleport?
 
-The Teleport creators used to work together at Rackspace. We noticed that most
-cloud computing users struggle with setting up and configuring infrastructure
-security because popular tools, while flexible, are complex to understand and
-expensive to maintain. Additionally, most organizations use multiple
-infrastructure form factors such as several cloud providers, multiple cloud
-accounts, servers in colocation, and even smart devices. Some of those devices
-run on untrusted networks, behind third party firewalls. This only magnifies
-complexity and increases operational overhead.
+The Teleport creators used to work together at Rackspace. We noticed that most cloud computing users struggle with setting up and configuring infrastructure security because popular tools, while flexible, are complex to understand and expensive to maintain. Additionally, most organizations use multiple infrastructure form factors such as several cloud providers, multiple cloud accounts, servers in colocation, and even smart devices. Some of those devices run on untrusted networks, behind third-party firewalls. This only magnifies complexity and increases operational overhead.
 
-We had a choice, either to start a security consulting business or build a
-solution that’s dead-easy to use and understand, something that creates an
-illusion of all of your servers being in the same room as you as if they were
-magically _teleported_. And Teleport was born!
+We had a choice, either start a security consulting business, or build a solution that's dead-easy to use and understand. A real-time representation of all of your servers in the same room as you, as if they were magically _teleported_. And Teleport was born!
```

Comment:
```suggestion
We had a choice, either start a security consulting business, or build a solution that's dead-easy to use and understand. A real-time representation of all of your servers in the same room as you, as if they were magically _teleported_. Thus, Teleport was born!
```

Reply:
Woo! I like the extra flair :) Nice touch.